scripts/rollup/build.js

```diff
@@ -150,6 +150,12 @@ function updateBundleConfig(config, filename, format, bundleType, hasteName) {
   });
 }
 
+function setUseFiberEnvVariable(useFiber) {
+  return {
+    'process.env.ROLLUP_USE_FIBER': useFiber,
```

Comment:
Could we have two versions of FeatureFlags, one with Fiber enabled and other with it disabled? And use aliasing to feed it either one.